solaris/tile/raster_tile.py

```diff
@@ -104,11 +102,11 @@ class RasterTiler(object):
         loaded.
     """
 
-    def __init__(self, dest_dir=None, dest_crs=None, channel_idxs=None,
-                 src_tile_size=(900, 900), src_metric_size=False,
+    def __init__(self, dest_dir=None, dest_crs=None, project_to_meters=False,
```

Comment:
I think this is the appropriate place to put this argument, but do want to note that it makes this a breaking change. I think we're going to push out 0.2.0 very soon, but we won't want to merge this change into `master` until we do.

Reply:
Makes sense!